Disable Dance Party tests on Safari Yosemite

islemaster opened the pull request:
These tests with a currently-known crash are slowing down test runs and generating a lot of noise.  I've [opened an issue](https://github.com/code-dot-org/dance-party/issues/406) on the dance-party repo to track this crash (which we haven't seen in the wild yet) and disabling the tests for now so our pipeline stays unblocked for other important changes.

davidsbailey commented on dashboard/test/ui/runner.rb:
```diff
@@ -609,6 +609,7 @@ def cucumber_arguments_for_browser(browser, options)
   # browser version 0 implies the latest version.
   arguments += skip_tag('@no_older_chrome') if browser['browserName'] == 'chrome' && (browser['version'].to_i != 0 && browser['version'].to_i <= 67)
   arguments += skip_tag('@no_safari') if browser['browserName'] == 'Safari'
+  arguments += skip_tag('@no_safariyosemite') if browser['browserName'] == 'Safari' && browser['version'] == '8.0'
```
is the browser version check to distinguish from mobile?  If so, it might be more robust to check `&& !browser['mobile']` or `&& browser['platform']&.starts_with?('OS X')`.

Alternately... I believe the existing `@no_safari` will already do what you want because of the case differences in browsers.json, although I haven't confirmed this.

islemaster replied:
Yeah, I noticed we're using `safari` on mobile and `Safari` on desktop but figured it'd be a bad idea to depend on that. :grin:  I suppose for a target called "SafariYosemite" it makes the most sense to say
```rb
browser['browserName'] == 'Safari' && browser['platform'] == 'OS X 10.10'
```
Pushing that change now.  Good idea!

islemaster commented:
Updated:
- Disabling the newest dance-party test `save_for_share.feature`.
- Replaced the `@no_safari` tag with `@no_safari_yosemite` everywhere. (Good idea Dave!)

davidsbailey commented:
I think this one would be good to merge today as well, out-of-band from the triage process.
